Composer/packages/client/src/pages/notifications/useNotifications.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { useContext, useMemo, useState } from 'react';
+
+import { StoreContext } from '../../store';
+import { createSingleMessage } from '../../utils/lgUtil';
+
+import { INotification } from './types';
+
+const DiagnosticSeverity = ['Error', 'Warning', 'Information', 'Hint'];
+
+export default function useNotifications() {
+  const { state } = useContext(StoreContext);
+  const [filter, setFilter] = useState('All');
```

Comment:
What are the different filters that can be set other than the string 'All' ? We should hoist these out and give it a type for better self-documentation if it's more like an enum.